.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -101,8 +95,22 @@ body:
     placeholder: What happened instead?
   validations:
     required: false
+    
+- id: additionalInfo
+  type: textarea
+  attributes:
+    label: Additional Information
+    placeholder: |
+      OS version
+      .Net version
+      Preferred System Language
```

Comment:
Personally, I don't like these words. "Preferred" implies the possibility that the choice is not implemented. And does it mean OS lang or App lang? (since PowerToys UI should be using the _app_ setting, right?)
It's just vague.

Reply:
Changed to "System Language"